ldcoolp/scripts/generate_wccfl_qualtrics_links

```diff
@@ -0,0 +1,224 @@
+    LD-Cool-P version: {__version__}
+    LD-Cool-P commit hash: {git_short_commit}
+    Created by Chun Ly
+    Issues? Submit a GitHub ticket: https://github.com/ualibraries/LD_Cool_P/issues/new
+    """
+
+    print(banner_message)
+
+    # Load configuration
+    config_dict = dict_load(args.config)
+
+    # Settings for script
+    config_dict['qualtrics']['survey_id'] = wccfl_survey_id
+
+    curation_dict = config_dict['curation']
+    root_directory_main = curation_dict[curation_dict['log_parent_dir']]
+
+    log_dir = join(root_directory_main, curation_dict['log_dir'])
+    if not exists(log_dir):
+        mkdir(log_dir)
+    logfile_prefix = 'generate_wccfl_qualtrics_links'
+    logfile = f'{logfile_prefix}.{today.strftime("%Y-%m-%d")}.log'
+
+    log = LogClass(log_dir, logfile).get_logger()
+
+    log.info("************************************")
+    log.debug(f"LD-Cool-P branch: {branch_name}")
+    log.debug(f"LD-Cool-P version: {__version__} ({git_short_commit})")
+    log.debug(f"LD-Cool-P commit hash: {git_commit}")
+
+    # Retrieve username, hostname, IP
+    sys_info = get_user_hostname()
+    log.debug(f"username : {sys_info['user']}")
+    log.debug(f"hostname : {sys_info['hostname']}")
+    log.debug(f"IP Addr  : {sys_info['ip']}")
+    log.debug(f"Op. Sys. : {sys_info['os']}")
+
+    # Configuration information
+    log.info(f"Config file: {args.config}")
+
+    # Read in CSV file
+    log.info(f"Reading: {args.csv_file}")
+    df = pd.read_csv(args.csv_file, sep='\t')
+    full_urls = [''] * len(df)
+    short_urls = [''] * len(df)
+
+    # Figshare API
+    ##############
+    fs_dict = config_dict['figshare']
+    fs_admin = figshare.FigshareInstituteAdmin(figshare_dict=fs_dict)
+
+    acct_df = fs_admin.get_account_list()
+    acct_dict = df_to_dict_single(acct_df.loc[acct_df['id'] == account_id])
+
+    log.info("Retrieving list now ...")
+
+    articles_df = get_articles_list(fs_dict)
+    log.info("Truncating to pending list from Depositor Account ...")
+    pending_df = articles_df.loc[articles_df['published_date'].isna()]
+    n_pending = len(pending_df)
+    log.info(f"Number of pending: {n_pending}")
+
+    pending_articles = pending_df['id'].tolist()
+
+    # Qualtrics API
+    ###############
+    q_dict = config_dict['qualtrics']
+
+    for p in range(len(pending_df)):
+        row = pending_df.loc[p]
+
+        article_dict = get_articles_list(fs_dict, article_id=row['id'])
+
+        # Match by title
+        match_idx = df.loc[df['title'] == article_dict['title']].index
+        match_df = df.loc[match_idx[0]]
+        dn_dict = {
+            'article_id': row['id'],
+            # 'curation_id': row['id'],
+            'title': article_dict['title'],
+        }
+        for key in ['full name', 'email', 'affiliation']:
+            dn_dict[key] = match_df[key]
+
+        full_url = wccfl_generate_url(q_dict, dn_dict)
+        full_urls[match_idx[0]] = full_url
+
+        if args.tiny_url:
```

Comment:
I think there were some changes recently related to TinyURL, so this part is not working correctly.